conan/cli/commands/config.py

```diff
@@ -77,3 +76,12 @@ def config_show(conan_api, parser, subparser, *args):
     args = parser.parse_args(*args)
 
     return conan_api.config.show(args.pattern)
+
+
+@conan_subcommand(formatters={"text": list_text_formatter, "json": default_json_formatter})
+def config_settings_yml(conan_api, parser, subparser, *args):
+    """
+    Get the content of the settings.yml file
+    """
+    parser.parse_args(*args)
+    return conan_api.config.settings_yml.possible_values()
```

Comment:
It might be useful, whenever I am checking for valid values, I don't want the full output. If not very readable on terminal, we might want to explore some syntax to select subsets.

Reply:
Yeah, I looked into that usecase when implementing it, but came to the conclusion as elsewhere that if `conan config settings-yml --format=json | jq .os` can do that for you, do we need to implement something for the text case too? I can totally pretty-print the text output, that's a good idea

Reply:
Sounds good, better than inventing a new argument or syntax